src/pkg/packaging/snaps/dotnet-runtime-2.1/snap/snapcraft.yaml

```diff
@@ -0,0 +1,44 @@
+name: dotnet-runtime-21
+version: 2.1.9
+summary: Cross-Platform .NET Core Runtime. 
+description: |
+  .NET Core 2.1 Runtime and ASP.NET Core. https://dot.net/core.
+
+grade: stable
+confinement: strict
+
+apps:
+  dotnet:
+    command: dotnet
+    plugs:
+      - network
+      - network-bind
+      - removable-media
+      - home
+
+slots:
+  dotnet-runtime:
+    content: dotnet-runtime-21
+    interface: content
+    read: [/]
+
+parts:
+  dotnet-runtime:
+      plugin: dump
```

Comment:
Nit: this uses 4 spaces rather than the 2 elsewhere.

Reply:
fixed